pkg/openSearch/openSearchClient/authenticator.go

```diff
@@ -52,7 +52,9 @@ func InjectAuthenticationIntoClient(client *opensearchapi.Client,
 	if err != nil {
 		return err
 	}
-	log.Debug().Msgf("Set up auth method for opensearch client: %s", method)
+	log.Debug().
+		Str("opensearch_auth_method", string(method)).
+		Msgf("set up auth method for opensearch client: %s", method)
```

Comment:
```suggestion
		Msg("set up auth method for opensearch client")
```
`auth_method` is already supplied separately

Reply:
Conclusion after discussion: 

The reason to keep the information in the log message itself, is that is makes it easier human readable in in Loki/Grafana: Less "drilling down" and not having to rely on individual fields in Loki, it's easier to read %+v


The log line should ideally have the non static part in the end, which is the case here.